sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/CheckpointStore/BlobsCheckpointStoreTests.cs

```diff
@@ -377,7 +576,7 @@ public void UpdateCheckpointForMissingCheckpointThrowsAndLogsCheckpointUpdateErr
 
             Assert.ThrowsAsync<RequestFailedException>(async () => await target.UpdateCheckpointAsync(checkpoint, new EventData(Array.Empty<byte>()), new CancellationToken()));
 
-            mockLog.Verify(m => m.CheckpointUpdateError(PartitionId, It.Is<string>(s => s.Contains(BlobErrorCode.ContainerNotFound.ToString()))));
+            mockLog.Verify(m => m.UpdateCheckpointError(PartitionId, FullyQualifiedNamespace, EventHubName, ConsumerGroup, It.Is<string>(s => s.Contains(BlobErrorCode.ContainerNotFound.ToString()))));
```

Comment:
nit: could simplify `It.Is<string>(s => s.Contains(BlobErrorCode.ContainerNotFound.ToString())` to `ex.Message`

Reply:
Yeah, I did a lot of copy/paste/ignore the buzzing in my brain in this file.   My intent was to be light of touch, though I ended up overhauling much more than I intended to.